docs/docs/gatsby-cli.md

```diff
@@ -385,24 +351,25 @@ Para mais informações, consulte a [documentação do REPL Gatsby](/docs/gatsby
 
   
 
-In addition to the explicit `--no-color` option, the CLI respects the presence of the `NO_COLOR` environment variable (see [no-color.org](https://no-color.org/)).
+Além da opção `--no-color`, o CLI respeita a utilização da variável de ambiente `NO_COLOR` (veja [no-color.org](https://no-color.org/)).
+
+## Como trocar o gerenciador de pacotes padrão no seu novo projeto?
 
-## How to change your default package manager for your next project?
+Quando você executa o comando `gatsby new` pela primeira vez para criar um novo projeto, ele pede para você escolher entre _yarn_ e _npm_ como gerenciador de pacotes padrão.
 
-When you use `gatsby new` for the first time to create a new project, you are asked to choose your default package manager between yarn and npm.
 
 ```shell
 Which package manager would you like to use ? › - Use arrow-keys. Return to submit.
 ❯  yarn
    npm
 ```
 
-Once you’ve made your choice, the CLI won’t ask for your preference again for any subsequent project.
+Uma vez que você escolhe, o ILC não ira perguntar a sua preferência nos prósimos projetos.
 
-If you want to change this for your next project you have to edit the config file created automatically by the CLI.
-This file is available on your system at: `~/.config/gatsby/config.json`
+Se você quiser trocar para essa configuração no seu próximo projeto, terá que editar o arquivo de configuração gerado automaticamente pelo ILC.
```

Comment:
```suggestion
Se você quiser trocar para essa configuração no seu próximo projeto, terá que editar o arquivo de configuração gerado automaticamente pelo CLI.
```

Acho que não precisa traduzir a sigla `CLI`, por que já é familiar na nossa área. Eu mesmo demorei um pouco pra sacar o que era `ILC` kkkk

Reply:
@henry-ns hahahah concordo contigo, mas é que tava seguindo essa planilha: https://docs.google.com/spreadsheets/d/1TLztI-jrPslCt0fgk4B9tGnhhS_7JQZJfMql1ePsIZo/edit#gid=0

Reply:
@henry-ns só precisava entender qual é o padrão, se for manter CLI aí seria legal atualizar a planilha e eu altero no arquivo inteiro.